karma/module.py

```diff
@@ -745,9 +745,10 @@ async def _process_reaction(
 
         if message is None:
             await guild_log.debug(
-                reaction.user_id,
-                reaction.channel_id,
-                f"Message {reaction.message_id} not found on karma reaction add.",
+                None,
+                None,
+                f"Message {reaction.message_id} not found on karma reaction add "
+                "by user {reaction.user_id} in channel {reaction.channel_id}.",
```

Comment:
This message will probably not be formatted.

Reply:
On line 750 there's no comman, so the 750 and 751 lines strings will be merged together to ` f"Message {reaction.message_id} not found on karma reaction add by user {reaction.user_id} in channel {reaction.channel_id}.",`

Reply:
Might there be some issues with formatting? It still feels like it will merge those strings, but the output will look like this: 
```
Message 2132132132132132 not found on karma reaction add by user {reaction.user_id} in channel {reaction.channel_id}.
```

```suggestion
                f"by user {reaction.user_id} in channel {reaction.channel_id}.",
```